sdk/servicebus/azure-servicebus/azure/servicebus/amqp/_amqp_message.py

```diff
@@ -43,6 +43,9 @@ def __ne__(self, other: Any) -> bool:
     def __str__(self) -> str:
         return str({k: v for k, v in self.__dict__.items() if not k.startswith("_")})
 
+    def __contains__(self, key):
```

Comment:
Looks like this mixin has been updated for type hints in Storage yet, but should be for ServiceBus.
```suggestion
    def __contains__(self, key: str) -> bool:
```